src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -121,6 +121,14 @@
 
     private static final Logger LOGGER = Logger.getLogger(WorkflowRun.class.getName());
 
+    private enum StopState {
+        TERM, KILL;
+
+        public String url() {
+            return this.name().toLowerCase();
```

Comment:
Must specify `Locale.ENGLISH` or Turkish and Azeri users will be left wondering why there is no `doKıll` method!